static/src/stylesheets/_mixins.scss

```diff
@@ -288,3 +288,15 @@
     &::-moz-placeholder { @content; }
     &:-ms-input-placeholder { @content; }
 }
+
+@function round-image-width($width) {
```

Comment:
maybe something like `snap-to-image-width` or something? probably something a bit terser than that, but you only get a width back (could be used for anything), and i'm not sure if it would be clear if you just came across it in what way it was being rounded?